Cargo.toml

```diff
@@ -3,7 +3,12 @@ name = "foxbox"
 version = "0.1.0"
 authors = ["fabrice <fabrice@desre.org>"]
 build = "build.rs"
-exclude = [ "config/**" ]
+# Files that shouldn't retrigger a new rust build if they are changed
+exclude = [ "config/**", "**/*.conf",
+            "static/**/*", "node_modules/**/*", "test/**/*", "tools/**/*", # Non-rust scripts
```

Comment:
Just to double check, this does not include changes to `static/shared`, right? Note that we need to run the build script every time something changes in this folder.